lexicons/com/atproto/admin/queryModerationStatuses.json

```diff
@@ -74,6 +74,10 @@
             "maximum": 100,
             "default": 50
           },
+          "langs": {
+            "type": "array",
+            "items": { "type": "string" }
+          },
```

Comment:
I think we could make these `flags` now, right? (maybe you were just focusing on the other lexicons for now)

Reply:
I thought the api seems a bit nicer if we used specifically named params like this that ultimately searches against flags. but now realizing that it would make things difficult to add more filters so will move this to flags instead.